### DISCUSSION TITLE
IgnoreLineBreak not working...

### DISCUSSION THREAD

**Discussion by testandattack** [Doraku/DefaultDocumentation#113] [General]
In my DefaultDocumentation.json file, I have:
```
  "Markdown": {
    "IgnoreLineBreak": true,
    "TableOfContentsModes": "IncludeKind,IncludeSummary"
  },

```
One of my XML blocks looks like this:

```
/// <summary>
/// Returns a list of strings from a single string that is CSV formatted. For more information, see
/// <see href="https://stackoverflow.com/questions/3776458/split-a-comma-separated-string-with-both-quoted-and-unquoted-strings">
/// this article</see>.
/// </summary>
/// <param name="source">The <c>string</c> to which this method is exposed.</param>
public static List<string> CsvStrToList(this string source)

```
But when I look at the output that is built, I still see the line breaks on some of the XML breaks. In this case, it appears to be related to using a `<see />` tag. Am I missing something?

![image](https://user-images.githubusercontent.com/86054303/172619156-74d1be0f-b933-4885-bdce-752184d493f7.png)

**Comment by Doraku** [comment]
Sorry for the delay! The actual properties in the configuration are `"Markdown.IgnoreLineBreak"` and `"Markdown.TableOfContentsModes"`, not properties of a sub object `"Markdown"` like you did in your configuration, that's why they are not used.

**Comment by testandattack** [comment]
Thanks for the reply. Just to be thorough, I tried the change you suggested, but that did not help. The format I was using appears to be valid since line breaks are being ignored when there is not a link included in the documentation. I am still able to reproduce the issue. Also, I noticed that the namespace page is built with some invalid markdown that causes it to break completely after the line in question. I hope I am simply doing something wrong, but if you get a chance, can you please look at [this page](https://github.com/testandattack/GTC.Extensions/blob/master/GTC.Extensions/MarkDown/GTC.Extensions.md#GTC.Extensions.StringExtensions). Thanks again, and many thanks for the utility. It is saving me a ton of time.

**Reply by Doraku** [reply]
ok I found the issue, a workaround would be to write your `<see>` as a single line in your doc obviously `<see href="https://stackoverflow.com/questions/3776458/split-a-comma-separated-string-with-both-quoted-and-unquoted-strings">this article</see>` (the important part is no linebreak inside the `<see>` element) as it is directly written as is ignoring the `MarkdownIgnoreLineBreak`) but I'll see to fix it, thanks for the infos you provided :)
